scripts/polynote.py

```diff
@@ -7,6 +7,10 @@
 if not os.environ.get('PYTHONPATH'):
     os.environ['PYTHONPATH'] = sys.prefix
 
+scala_version = os.environ.get('POLYNOTE_SCALA_VERSION')
```

Comment:
You can provide a default value: `os.environ.get('POLYNOTE_SCALA_VERSION', '2.11')`